src/libs/deploy.js

```diff
@@ -135,8 +136,7 @@ export const deployGoogle = async (signedApp, workspace = '/tmp/') => {
     // Get the Google client-service key to deployment:
     const keyFull = await exec(`security find-generic-password -w -s deployKey -a ${apkBundleId}`);
     const keyPath = keyFull.stdout.trim().split('\n');
-    // eslint-disable-next-line import/no-dynamic-require
-    const key = require(`${keyPath}`); // TODO:(jl) This require should go.
+    const key = JSON.parse(fs.readFileSync(`${keyPath}`));
```

Comment:
You don't need string templates here, just use keyPath.

Reply:
keyPath is pointing to a file locally. Can't read content directly...